src/Std/Time/Duration.lean

```diff
@@ -0,0 +1,339 @@
+/-
+Copyright (c) 2024 Lean FRO, LLC. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Sofia Rodrigues
+-/
+prelude
+import Std.Time.Date
+import Std.Time.Time
+
+namespace Std
+namespace Time
+open Internal
+
+set_option linter.all true
+
+/--
+Represents a point in time relative to the UNIX Epoch, with nanoseconds precision.
```

Comment:
I think this docstring is not correct. Also, couldn't you just define `Duration` as `Nanosecond.Offset`?

Reply:
Okay, let's leave it like this.